File: WebRoot/app/materialInspection/model/MaterialInspection.js
Ext.define('erp.materialInspection.model.MaterialInspection', {header:'',
	extend: 'Ext.data.Model',
	idProperty: '',
	fields: [
	    {header:'签发',name: 'qfbj', type: 'int' ,columnWidth:40,isSign:true},
		{header:'检验',name: 'jybj', type: 'int' ,columnWidth:40,isSign:true},
		{header:'锁定',name: 'sdbj', type: 'int' ,columnWidth:40,isSign:true},
		{header:'委托单号',name: 'wtdh', type: 'int' ,columnWidth:80},
		{header:'委托部门',name: 'wtlbmc' ,columnWidth:80},
		{header:'委托类别',name: 'wtlb' ,columnWidth:80,hidden:true},
		{header:'委托人',name: 'wtrm' ,columnWidth:80},
		{header:'委托日期',name: 'wtrq', type: 'date', dateFormat: 'Y-m-d H:i:s' ,columnWidth:90},
		{header:'供应厂商',name: 'csmc' ,columnWidth:200},
		{header: '测试类型',name:'cslx',columnWidth:80},
		{header: '测试机构',name:'csjgmc',columnWidth:180},
		{header: '测试机构',name:'csjg',columnWidth:80,hidden:true},
	    {header:'序号',name: 'wtxh', type: 'int' ,columnWidth:60},
	    
	    {header:'材料货号', name: 'clhh',columnWidth:80},
	    {header:'材料名称', name: 'clmc',columnWidth:180},
	    {header:'送检数量',name: 'sjsl', type: 'float' ,columnWidth:80,align:'center'},
		{header:'单位',name: 'jldw' ,columnWidth:60},
		{name: 'htbh', type: 'float',hidden:true},
		{name: 'htxh', type: 'float',hidden:true},
		{name: 'csbh',hidden:true },
		{name: 'cghtyq',hidden:true },
		{header:'合同号',name: 'hth' ,columnWidth:80},
		{header:'订单号',name: 'ddh' ,columnWidth:80},
		{header:'来料日期',name: 'llrq', type: 'date', dateFormat: 'Y-m-d H:i:s' ,columnWidth:90},
		{header:'测试目的',name: 'csmd' ,columnWidth:200},
		{header:'备注说明',name: 'bzsm' ,columnWidth:200},
		{header:'操作员',name: 'czym' ,columnWidth:70},
		{header:'操作时间',name: 'czsj', type: 'date', dateFormat: 'Y-m-d H:i:s' ,columnWidth:90},
		{header:'锁定人名',name: 'sdrm' ,columnWidth:70},
		{header:'锁定时间',name: 'sdsj', type: 'date', dateFormat: 'Y-m-d H:i:s' ,columnWidth:80},
		 {header:'签发人名',name: 'qfrm' ,columnWidth:80},
		{header:'签发时间',name: 'qfsj', type: 'date', dateFormat: 'Y-m-d H:i:s' ,columnWidth:80},
		
		{name: 'jybj', type: 'int' ,hidden:true},
		{name: 'gdbj', type: 'int' ,hidden:true},
		{name: 'gdbj_pd', type: 'int' ,hidden:true},
		
		{name: 'shbj', type: 'int' ,hidden:true},
		{name: 'ckbh' ,hidden:true},
		{name: 'dhdh', type: 'float' ,hidden:true},
		{name: 'dhxh', type: 'float' ,hidden:true},
		{name: 'bzsm' ,hidden:true},
		{name: 'jysj', type: 'date', dateFormat: 'Y-m-d H:i:s' ,hidden:true},
		{name: 'shsj', type: 'date', dateFormat: 'Y-m-d H:i:s' ,hidden:true},
		{name: 'csmd' ,hidden:true},
		{name: 'jjcd' ,hidden:true}
	]
});

File: src/erp/erp/materialInspection/model/MaterialInspection.java
package erp.erp.materialInspection.model;

import java.io.Serializable;
import java.util.Date;

import erp.common.Model;

public class MaterialInspection extends Model implements Serializable{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private float			wtdh;
	private float			wtxh;
	private String			wtbm;
	private String			clhh;
	private String			wtlb;
	private String			wtrm;
	private String			hth;
	private String			ddh;
	private Date			wtrq;
	private float			htbh;
	private float			htxh;
	private String			csbh;
	private Date			llrq;
	private float			sjsl;
	private String			jldw;
	private int			sdbj;
	private String			sdrm;
	private Date			sdsj;
	private int			jybj;
	private int			gdbj;
	private int			gdbj_pd;
	private String			czym;
	private Date			czsj;
	private int			shbj;
	private String			ckbh;
	private float			dhdh;
	private float			dhxh;
	private String			bzsm;
	private String			wth;
	private Date			jysj;
	private Date			shsj;
	private String			csmd;
	private String			csjg;
	private String			cslx;
	private String			clmc;
	private String			csmc;
	private String			wtlbmc;
	private String			csjgmc;
	private String			plmth;
	private String			plmtx;
	private String			checked;
	private String			cghtyq;
	private int			qfbj;
	private String			qfrm;
	private Date			qfsj;
	private String			jjcd;
    private Integer          id;

    
	public Integer getId() {
		return id;
	}
	public void setId(Integer id) {
		this.id = id;
	}
	public String getCghtyq() {
		if(cghtyq==null){
			return "";
		}
		return cghtyq;
	}
	public void setCghtyq(String cghtyq) {
		this.cghtyq = cghtyq;
	}
	public String getWth() {
		return wth;
	}
	public void setWth(String wth) {
		this.wth = wth;
	}
	public String getChecked() {
		return checked;
	}
	public void setChecked(String checked) {
		this.checked = checked;
	}
	public String getCsjgmc() {
		return csjgmc;
	}
	public void setCsjgmc(String csjgmc) {
		this.csjgmc = csjgmc;
	}
	public String getWtlbmc() {
		return wtlbmc;
	}
	public void setWtlbmc(String wtlbmc) {
		this.wtlbmc = wtlbmc;
	}
	public String getCsmc() {
		return csmc;
	}
	public void setCsmc(String csmc) {
		this.csmc = csmc;
	}
	public float getWtxh() {
		return wtxh;
	}
	public void setWtxh(float wtxh) {
		this.wtxh = wtxh;
	}
	public String getHth() {
		return hth;
	}
	public void setHth(String hth) {
		this.hth = hth;
	}
	public String getDdh() {
		return ddh;
	}
	public void setDdh(String ddh) {
		this.ddh = ddh;
	}
	public String getClmc() {
		return clmc;
	}
	public void setClmc(String clmc) {
		this.clmc = clmc;
	}
	public String getPlmth() {
		if(plmth==null){
			return "";
		}
		return plmth;
	}
	public void setPlmth(String plmth) {
		this.plmth = plmth;
	}
	public String getPlmtx() {
		if(plmtx==null){
			return "";
		}
		return plmtx;
	}
	public void setPlmtx(String plmtx) {
		this.plmtx = plmtx;
	}
	public float getWtdh() {
		return wtdh;
	}
	public void setWtdh(float wtdh) {
		this.wtdh = wtdh;
	}
	public String getWtbm() {
		if(wtbm==null){
			return "";
		}
		return wtbm;
	}
	public void setWtbm(String wtbm) {
		this.wtbm = wtbm;
	}
	public String getWtlb() {
		if(wtlb==null){
			return "";
		}
		return wtlb;
	}
	public void setWtlb(String wtlb) {
		this.wtlb = wtlb;
	}
	public String getWtrm() {
		if(wtrm==null){
			return "";
		}
		return wtrm;
	}
	public void setWtrm(String wtrm) {
		this.wtrm = wtrm;
	}
	public Date getWtrq() {
		return wtrq;
	}
	public void setWtrq(Date wtrq) {
		this.wtrq = wtrq;
	}
	public String getClhh() {
		if(clhh==null){
			return "";
		}
		return clhh;
	}
	public void setClhh(String clhh) {
		this.clhh = clhh;
	}
	public float getHtbh() {
		return htbh;
	}
	public void setHtbh(float htbh) {
		this.htbh = htbh;
	}
	public float getHtxh() {
		return htxh;
	}
	public void setHtxh(float htxh) {
		this.htxh = htxh;
	}
	public String getCsbh() {
		if(csbh==null){
			return "";
		}
		return csbh;
	}
	public void setCsbh(String csbh) {
		this.csbh = csbh;
	}
	public Date getLlrq() {
		return llrq;
	}
	public void setLlrq(Date llrq) {
		this.llrq = llrq;
	}
	public float getSjsl() {
		return sjsl;
	}
	public void setSjsl(float sjsl) {
		this.sjsl = sjsl;
	}
	public String getJldw() {
		if(jldw==null){
			return "";
		}
		return jldw;
	}
	public void setJldw(String jldw) {
		this.jldw = jldw;
	}
	public int getSdbj() {
		return sdbj;
	}
	public void setSdbj(int sdbj) {
		this.sdbj = sdbj;
	}
	public String getSdrm() {
		if(sdrm==null){
			return "";
		}
		return sdrm;
	}
	public void setSdrm(String sdrm) {
		this.sdrm = sdrm;
	}
	public Date getSdsj() {
		return sdsj;
	}
	public void setSdsj(Date sdsj) {
		this.sdsj = sdsj;
	}
	public int getJybj() {
		return jybj;
	}
	public void setJybj(int jybj) {
		this.jybj = jybj;
	}
	public int getGdbj() {
		return gdbj;
	}
	public void setGdbj(int gdbj) {
		this.gdbj = gdbj;
	}
	public int getGdbj_pd() {
		return gdbj_pd;
	}
	public void setGdbj_pd(int gdbj_pd) {
		this.gdbj_pd = gdbj_pd;
	}
	public String getCzym() {
		if(czym==null){
			return "";
		}
		return czym;
	}
	public void setCzym(String czym) {
		this.czym = czym;
	}
	public Date getCzsj() {
		return czsj;
	}
	public void setCzsj(Date czsj) {
		this.czsj = czsj;
	}
	public int getShbj() {
		return shbj;
	}
	public void setShbj(int shbj) {
		this.shbj = shbj;
	}
	public String getCkbh() {
		if(ckbh==null){
			return "";
		}
		return ckbh;
	}
	public void setCkbh(String ckbh) {
		this.ckbh = ckbh;
	}
	public float getDhdh() {
		return dhdh;
	}
	public void setDhdh(float dhdh) {
		this.dhdh = dhdh;
	}
	public float getDhxh() {
		return dhxh;
	}
	public void setDhxh(float dhxh) {
		this.dhxh = dhxh;
	}
	public String getBzsm() {
		if(bzsm==null){
			return "";
		}
		return bzsm;
	}
	public void setBzsm(String bzsm) {
		this.bzsm = bzsm;
	}
	public Date getJysj() {
		return jysj;
	}
	public void setJysj(Date jysj) {
		this.jysj = jysj;
	}
	public Date getShsj() {
		return shsj;
	}
	public void setShsj(Date shsj) {
		this.shsj = shsj;
	}
	public String getCsmd() {
		if(csmd==null){
			return "";
		}
		return csmd;
	}
	public void setCsmd(String csmd) {
		this.csmd = csmd;
	}
	public String getCsjg() {
		if(csjg==null){
			return "";
		}
		return csjg;
	}
	public void setCsjg(String csjg) {
		this.csjg = csjg;
	}
	public String getCslx() {
		if(cslx==null){
			return "";
		}
		return cslx;
	}
	public void setCslx(String cslx) {
		this.cslx = cslx;
	}
	public int getQfbj() {
		return qfbj;
	}
	public void setQfbj(int qfbj) {
		this.qfbj = qfbj;
	}
	public String getQfrm() {
		if(qfrm==null){
			return "";
		}
		return qfrm;
	}
	public void setQfrm(String qfrm) {
		this.qfrm = qfrm;
	}
	public Date getQfsj() {
		return qfsj;
	}
	public void setQfsj(Date qfsj) {
		this.qfsj = qfsj;
	}
	public String getJjcd() {
		if(jjcd==null){
			return "";
		}
		return jjcd;
	}
	public void setJjcd(String jjcd) {
		this.jjcd = jjcd;
	}
}
